deepseek/test.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+import time
+from openai import OpenAI
+
+# Set your DeepSeek API key here
+isana_api_key = 'isana_api_key'
+
+client = OpenAI(
+    # This is the default and can be omitted
+    api_key=isana_api_key,
+    base_url="https://api.deepseek.com"
+)
+
+start_time = time.time()
+
+#chat_completion = client.chat.completions.create(
+response = client.chat.completions.create(
+    messages=[
+        {"role": "system", "content": "You are an experienced data engineer."},
+        {
+            "role": "user",
+            "content": "What are the NoSQL options to store JSON documents?"
+        }
+    ],
+#    model="deepseek-chat",
+    model="deepseek-coder",
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Potential mismatch in model selection</b></div>
<div id="fix">

The model being used is `deepseek-coder`. It might be worth considering if this is the most appropriate model for the task of answering questions about NoSQL options for storing JSON documents.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -27,1 +27,1 @@
 -    model="deepseek-coder",
 +    model="deepseek-chat",
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/adhir-potdar/llm-connect/pull/1#issuecomment-2451596739>#ce0fd9</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged